src/gui/StringPairDrag.cpp

```diff
@@ -80,37 +80,30 @@ StringPairDrag::~StringPairDrag()
 
 
 
-bool StringPairDrag::processDragEnterEvent( QDragEnterEvent * _dee,
-						const QString & _allowed_keys )
+bool StringPairDrag::processDragEnterEvent(QDragEnterEvent * _dee,
+						const QString & _allowed_keys)
```

Comment:
Instead of a comma-delimited string of allowed keys, maybe you could use `std::initializer_list<QString>`?

Then calling code could use
```cpp
StringPairDrag::processDragEnterEvent(dee, {
	"samplefile",
	QString{"clip_%1"}.arg(static_cast<int>(Track::Type::Sample))
});
```

Ideally we'd use `std::span<QString>` but unfortunately that can't be constructed from a `std::initializer_list<QString>` until C++26.